src/pages/index.js

```diff
@@ -251,31 +247,28 @@ export default () => {
       <FlexContainer>
         <Image src={strategy} />
         <Advantage>
-          <SectionHeader>Manage complexity like a boss</SectionHeader>
+          <SectionHeader>Universal code</SectionHeader>
           <p>
-            Testable, statically typed primitives that don't get in your way
+            Write your server and client code together and Fusion will take care of the rest.
           </p>
           <Snippet>
             <pre
               // eslint-disable-next-line react/no-danger
               dangerouslySetInnerHTML={{
                 __html: highlight(`
-/* @flow */
+import App from 'fusion-react';
+import SecureHeadersPlugin from './plugins/secure-headers-plugin';
 
-import {createPlugin, createToken} from 'fusion/core';
-import {UserServiceToken as User} from './user';
-import {ProjectServiceToken as Project} from './project';
+export default () => {
+  const app = new App(<div>...</div>);
 
-export const TeamServiceToken = createToken('TeamServiceToken');
-export const TeamService = createPlugin({
-  deps: {User, Project},
-  provides: ({User, Project}) => ({
-    async findByProjectId(projectId) {
-      const {users} = await Project.findById(projectId);
-      return Promise.all(users.map(id => User.findById(id)));
-    },
-  }),
-});
+  // Register this plugin only on the server
+  if (__NODE__) {
+    app.register(SecureHeadersPlugin);
```

Comment:
I think that maybe `SecureHeadersPlugin` might confuse newcomers. I wonder  if we just have a simple inline plugin  that logs the current route or something.

Reply:
I just picked something that sounded server side’y and that wouldn’t be applicable on the client. Inlining code would add to the example code length also which I’m trying to avoid since the point of this section is about code fencing and not plugin code.

That being said I could maybe change it to something less jargony but I think SecureHeaders hopefully isn’t too complex of a term.